Neos.Neos/Classes/Fusion/AbstractMenuItemsImplementation.php

```diff
@@ -84,7 +84,7 @@ public function getItems()
     {
         if ($this->items === null) {
             $fusionContext = $this->runtime->getCurrentContext();
-            $this->currentNode = isset($fusionContext['activeNode']) ? $fusionContext['activeNode'] : $fusionContext['documentNode'];
+            $this->currentNode = $fusionContext['activeNode'] ?? $fusionContext['documentNode'];
```

Comment:
`activeNode` is not used anywhere, and it seems it never even was. Can it go? 🤷‍♂️

Reply:
I would suggest to split the actual bugfix from all other cleanups tbh, those can go into `master`